app/assets/javascripts/auto_title.js

```diff
@@ -0,0 +1,24 @@
+function onTitleRetrievalSuccess(data, status, xhr) {
+  // $(#bookmark_title).val(data);
```

Comment:
Also `data` here is going to be JSON and not simply a string you can plug straight into a DOM element.